schema.sql

```diff
@@ -0,0 +1,17 @@
+CREATE DATABASE cheesecake;
```

Comment:
This is the correct file type. I think we should iterate a bit more on our tables though.